town.logic

```diff
@@ -1,2 +1,44 @@
 area Town:
-  room Main
\ No newline at end of file
+  room Main:
+    node Main:
+      chest CafeLady
+      door Bakery
+      door East
+      door FigurineHouseEast
```

Comment:
Accessing the figurine house from the east requires Carlov Medal, so it should be on its own connected node.

Reply:
Fixed it!